data/transactions/logic/evalStateful_test.go

```diff
@@ -2322,22 +2317,22 @@ func TestReturnTypes(t *testing.T) {
 	ledger.NewLocal(tx.Receiver, 1, string(key), algoValue)
 	ledger.NewAccount(appAddr(1), 1000000)
 
+	// We try to form a snippet that will test every opcode, by sandwiching it
```

Comment:
@jannotti While improving my comprehension about the test, I opted to provide an optional refactoring:  https://github.com/jannotti/go-algorand/pull/4.  Can you take a look?

If you'd prefer to keep as is, feel welcomed to close the PR.  While I feel the PR _is_ an improvement, I don't feel strongly.

Reply:
merged

Reply:
@jannotti Thanks for bringing in the PR.